src/main/java/com/robin/hoteless/Reservation.java

```diff
@@ -0,0 +1,42 @@
+package com.robin.hoteless;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+public class Reservation {
+    int id;
+    Date made;
+    int stayDays;
+    double dailyRate;
+    LocalDate in;
+    LocalDate out;
+    String bedType;
+    double total;
+    static int totalReservations;
+
+    Reservation(Date m, int s, LocalDate i, LocalDate o, double d, String b, double t) {
```

Comment:
Try to avoid single letter variable names, as simple as they seem; it will wreak havoc on debugging at later dates ;) Try to always give variable names specific and memorable names.

Reply:
Also, though we chatted about this already, these variable names should match the field definitions above exactly. Then make ``this.stayDays = stayDays`` as standard notation.